grumblr/app/controllers/application_controller.rb

```diff
@@ -2,4 +2,14 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-end
+
+    def set_access_control_headers
```

Comment:
We don't need this method technically. Since our JS (backbone) is served from the same server (rails), we don't need to enable CORS.